packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -243,15 +252,13 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     sourceChannel: agToNoble.transferChannel.counterPartyChannelId,
     remoteDenom: 'uusdc',
     settlementAccount,
-    intermediateRecipient,
   });
 
-  const advancer = zone.makeOnce('Advancer', () =>
-    makeAdvancer({
+  const { advancer, admin: advancerAdmin } = zone.makeOnce('AdvancerKit', () =>
```

Comment:
while separate an `admin` facet when `advancer` is never passed out of this scope? It's also exposing two advancer facets that weren't before. Please justify (perhaps in a comment) or revert

Reply:
after internal discussion, I realized operating with no noble connection is change to the advancer API, so we might as well put the new method there.